.goreleaser.yml

```diff
@@ -0,0 +1,26 @@
+before:
+  hooks:
+    - go mod download
+    - go generate ./...
+builds:
+- env:
+  - CGO_ENABLED=0
+- goos:
+  - darwin
+  - linux
+  - windows
+- goarch:
+  - amd64
+archives:
+- replacements:
+    amd64: x86_64
+checksum:
+  name_template: 'checksums.txt'
+snapshot:
+  name_template: "{{ .Tag }}-next"
```

Comment:
What does this do?

Reply:
goreleaser supports a [snapshot functionality](https://goreleaser.com/customization/snapshots/) to allow you to test your CI pipeline. This just allows us to template out the name of the snapshot. In this configuration, it output will be similar to this: `pathogen_v0.2.0-SNAPSHOT_linux_x86_64.tar.gz`

Here is blurb from the page:

> Note that the idea behind GoReleaser's snapshots if mostly for local builds or to validate your build on the CI pipeline. Artifacts shouldn't be uploaded anywhere, and will only be generated to the dist folder.

I'm using this to add a make target to allow you to test locally.